Makefile

```diff
@@ -33,13 +34,25 @@ APP_SRC_BUCKET = $(NAMESPACE)-app-dist
 TERRAFORM_DIR = terraform
 export BOOTSTRAP_ENV=terraform/bootstrap
 
+ifeq ($(ENV_NAME), prod)
+DOMAIN=
```

Comment:
Is there any risk for having this variable blank on prod

Reply:
nope, blank value configures terraform and no dns + ssl association will be created.